### ISSUE TITLE
Too many items being created

### ISSUE THREAD

**Issue by 128bitengine** [Nanite-Construction-System/NaniteConstructionSystem#99]
Currently my factory creates at least 10X everything it needs. It is creating 3.1k solar cells for 4 solar panels. This is the same for all blocks.

**Comment by nukeguard** [comment]
Can you reproduce this same effect in a new world with facility and assembler? With only nanites mod in the world?

**Comment by 128bitengine** [comment]
Yes. I spent about 6 to 8 hours making an entire new factory from scratch (so much cobalt ore) and I still have these issues. I noticed that the mod appeared to be calculating the needs of the ship, and then multiplying them by 5x or so before resetting on the proper number.  I attached some screen shots of the numbers going up. In calc1, it has the proper numbers, and in calc 2 and 3 you see it increasing, it then goes back to calc1's numbers and sits for a few seconds before beginning the scan again.  If these numbers get high, and then the mod orders the parts to be built before it resets, this could be why this is overloading the assemblers with unneeded parts

![calc1](https://user-images.githubusercontent.com/42009718/53289567-23958b00-375d-11e9-9fe6-3f92eb82bb61.jpg)
![calc2](https://user-images.githubusercontent.com/42009718/53289570-27291200-375d-11e9-98c2-c7ff0a7efeea.jpg)
![calc3](https://user-images.githubusercontent.com/42009718/53289571-28f2d580-375d-11e9-9260-05e140e826be.jpg)
